### DISCUSSION TITLE
How can I enable vertical scroll in the Filament Rich Editor?

### DISCUSSION THREAD

**Discussion by rjalonso1980** [filamentphp/filament#11968] [General]
Hi friends, 

I implemented the Filament Rich Editor and I have the following issue. When users use the editor and write very long texts, they have to scroll on the browser screen to use the toolbar (e.g. add a link).
Is there a way to add a vertical scroll to the editor? Or make the toolbar sticky?

The problem I have is that the CSS is compiled in production and I can't know the names of the classes to override their style.
Can you help me? 

Thank you!

**Comment by yuvrajtimalsina** [comment]
Chain this to your RichEditor

```
 ->extraAttributes(['class' => 'max-h-96'])
 ```

**Comment by meharenderb** [comment]
I added this and it worked for me without touching any core css of filament.

`->extraInputAttributes(['class' => 'max-h-96', 'style' => 'overflow-y: scroll;'])`

**Comment by iranvel** [comment]
->maxHeight('400px')
